test/unit/workers/check_organisation_links_worker_test.rb

```diff
@@ -23,15 +23,59 @@ class CheckOrganisationLinksWorkerTest < ActiveSupport::TestCase
   end
 
   test "when given an organisation it only calls LinkCheckerApiService with editions for that organisation" do
-    stub_published_publication
+    checked_edition = stub_published_publication
```

Comment:
Should this really be a `non_checked_edition`? As I understand it, it's a new edition that hasn't been checked yet?

Reply:
the variable probably needs a better name :)